modules/cats-effect/src/main/scala/pureconfig/module/catseffect/package.scala

```diff
@@ -6,24 +6,19 @@ import java.nio.file.Path
 import scala.language.higherKinds
 import scala.reflect.ClassTag
 
-import cats.data.NonEmptyList
-import cats.effect.Sync
+import cats.effect.{ Blocker, ContextShift, Sync }
 import cats.implicits._
-import com.typesafe.config.{ ConfigRenderOptions, Config => TypesafeConfig }
+import com.typesafe.config.ConfigRenderOptions
 import pureconfig._
 import pureconfig.error.ConfigReaderException
 
 package object catseffect {
 
-  @deprecated("Root will be treated as the default namespace", "0.12.0")
```

Comment:
We have no fixed rule to delete deprecated methods, but we usually take way more than one breaking version to delete them. Can you please revert all deletions?